qa/rpc-tests/fundrawtransaction.py

```diff
@@ -28,7 +28,11 @@ def setup_network(self, split=False):
 
     def run_test(self):
         print "Mining blocks..."
-        feeTolerance = Decimal(0.00000002) #if the fee's positive delta is higher than this value tests will fail, neg. delta always fail the tests
+
+        min_relay_tx_fee = self.nodes[0].getnetworkinfo()['relayfee']
+        # if the fee's positive delta is higher than this value tests will fail, neg. delta always fail the tests
```

Comment:
Can you update this comment to explain why this is picked (signatures may or may not take an extra byte or two, but never be smaller)